src/nomad_simulations/properties/energies.py

```diff
@@ -77,3 +78,502 @@ def __init__(
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+##########################
+# Other / General energies
+##########################
+
+
+
+    # madelung = SubSection(
+    #     sub_section=EnergyEntry.m_def,
+    #     description="""
+    #     Contains the value and information regarding the Madelung energy.
+    #     """,
+    # )
+
+    # # TODO I suggest ewald is moved to "long range" under electrostatic->energyentry, unless there is some other usage I am misunderstanding
+    # ewald = SubSection(
+    #     sub_section=EnergyEntry.m_def,
+    #     description="""
+    #     Contains the value and information regarding the Ewald energy.
+    #     """,
+    # )
+
+    # free = SubSection(
+    #     sub_section=EnergyEntry.m_def,
+    #     description="""
+    #     Contains the value and information regarding the free energy (nuclei + electrons)
+    #     (whose minimum gives the smeared occupation density calculated with
+    #     smearing_kind).
+    #     """,
+    # )
+
+    # sum_eigenvalues = SubSection(
+    #     sub_section=EnergyEntry.m_def,
+    #     description="""
+    #     Contains the value and information regarding the sum of the eigenvalues of the
+    #     Hamiltonian matrix.
+    #     """,
+    # )
+
+    # van_der_waals = SubSection(
+    #     sub_section=EnergyEntry.m_def,
+    #     description="""
+    #     Contains the value and information regarding the Van der Waals energy. A multiple
+    #     occurence is expected when more than one van der Waals methods are defined. The
+    #     van der Waals kind should be specified in Energy.kind
+    #     """,
+    # )
+
+    # hartree_fock_x_scaled = SubSection(
+    #     sub_section=EnergyEntry.m_def,
+    #     description="""
+    #     Scaled exact-exchange energy that depends on the mixing parameter of the
+    #     functional. For example in hybrid functionals, the exchange energy is given as a
+    #     linear combination of exact-energy and exchange energy of an approximate DFT
+    #     functional; the exact exchange energy multiplied by the mixing coefficient of the
+    #     hybrid functional would be stored in this metadata. Defined consistently with
+    #     XC_method.
+    #     """,
+    # )
+
+    # ? This is technically NOT redundant with total energy, but I fear that people will use them interchangeably, so we need to be clear about the definitions in any case
+    # internal = Quantity(
+    #     type=np.dtype(np.float64),
+    #     shape=[],
+    #     unit='joule',
+    #     description="""
+    #     Value of the internal energy.
+    #     """,
+    # )
+
+    # double_counting = SubSection(
+    #     sub_section=EnergyEntry.m_def,
+    #     categories=[FastAccess],
+    #     description="""
+    #     Double counting correction when performing Hubbard model calculations.
+    #     """,
+    # )
```

Comment:
While not a physical energy, per se, it is a term in the Hamiltonian used for corrections. This begs the question: do we want to distinguish Hamiltonian terms or physical phenomena?

Reply:
Just repeating myself from above: We should definitely NOT store the Hamiltonian terms here. That goes in the method. However, we may store the evaluation of the Hamiltonian for particular configurations of particles. This should be more clear though for sure.

Reply:
> Just repeating myself from above: We should definitely NOT store the Hamiltonian terms here

I agree with you (see above), but I have the impression that you were doing that anyhow here.